wormhole/wormhole_test.go

```diff
@@ -754,6 +754,25 @@ func TestClient_relayURL_default(t *testing.T) {
 	}
 }
 
+// if TransitRelayAddress is set, then it is used instead of
+// TransitRelayURL
+func TestClient_relayURL_relayAddress(t *testing.T) {
+	var c Client
+
+	// transitRelayAddress is in host:port format
+	// protocol is assumed as "tcp".
+	c.TransitRelayAddress = "transit.magic-wormhole.io:4001"
+
+	// if proto field is empty in the input string, then
+	// relayURL() would deduce it as "tcp".
+	url := c.relayURL()
+
+	expectedProto := "tcp"
+	if url.Proto != expectedProto {
+		t.Error(fmt.Sprintf("invalid protocol, expected %v, got %v", expectedProto, url.Proto))
+	}
+}
```

Comment:
I like to use [this assertion library](https://github.com/stretchr/testify) (specifically `require`) to save some time and keystrokes. I would also add some assertions which might make it make sense to refactor a bit.

I might do something like this:
```golang
require.Equal(t, expectedProto, url.Proto)
require.Equal(t, expectedHost, url.Host)
require.Equal(t, expectedPort, url.Port)
require.Equal(t, c.TransitRelayAddress, url.String())
```

Maybe we could do something similar for the test above this one as well: [TestClient_relayURL_default](https://github.com/LeastAuthority/wormhole-william/blob/fdb37350b7952fec97e048d2eeae7f6656d32628/wormhole/wormhole_test.go#L747)

Reply:
Good idea.

I suggest we do that for the entire tests as a separate PR for consistency and leave it for now and track it with a ticket?